src/debug.c

```diff
@@ -477,7 +477,7 @@ NULL
         };
         addReplyHelp(c, help);
     } else if (!strcasecmp(c->argv[1]->ptr,"segfault")) {
-        raise(SIGSEGV);
+        *((char*)-1) = 'x';
```

Comment:
This seems weird to me. Why no raise a segfault explicitly? Writing to address -1 wouldn't necessarily do this, or am I wrong?